src/invidious/views/watch.ecr

```diff
@@ -35,11 +35,11 @@
 <h1>
     <%= HTML.escape(video.title) %> 
     <% if params[:listen] %>
-    <a href="/watch?<%= env.params.query %>&listen=0">
+    <a title="Video/audio mode" href="/watch?<%= env.params.query %>&listen=0">
```

Comment:
I'd probably just call this "Video mode", with audio being implied.

Reply:
Done, also take the opportunity to make some minor fixes (details are in the commit).